MekHQ/src/mekhq/gui/StratconPanel.java

```diff
@@ -371,6 +420,51 @@ private boolean drawHexes(Graphics2D g2D, DrawHexType drawHexType) {
 
         return pointFound;
     }
+    
+    /**
+     * Retrieves a buffered image from a file given a key into the config file (StratconBiomeManifest.xml)
+     */
+    private BufferedImage getImage(String imageKey, ImageType imageType) {
+        if (imageCache.containsKey(imageKey)) {
+            return imageCache.get(imageKey);
+        }
+        
+        String imageName = null;
+        
+        switch (imageType) {
+            case TerrainTile:
+                imageName = StratconBiomeManifest.getInstance().getBiomeImage(imageKey);
+                break;
+            case Facility:
+                imageName = StratconBiomeManifest.getInstance().getFacilityImage(imageKey);
+                break;
+        }
+
+        if (imageName == null) {
+            return null;
+        }
+
+        File biomeImageFile = new File(imageName);
+        BufferedImage image = null;
+
+        try {
+            image = ImageIO.read(biomeImageFile);
+        } catch (Exception e) {
+            LogManager.getLogger().error("Unable to load image: " + imageName + " with ID '" + imageKey + "'");
+            return null;
+        }
+        
+        double xScale = HEX_X_RADIUS * 2.0 / image.getWidth();
```

Comment:
Maybe ImageUtil.getScaledImage() can be used here. There's a version where you can set the scale filter type.

Reply:
Done.